src/pages/Register.jsx

```diff
@@ -29,12 +29,15 @@ const Register = ({ authPopup }) => {
         password: "",
         password2: "",
         termsAndConditions: "",
+        // this is intialized with 'true' as the checkbox is checked upon page render
+        notifications: "true",
```

Comment:
it is a boolean type and not string so you need to change to
`notifications: true`

Reply:
@ArkadiK94 Silly me! I'm on it.